event_routing_backends/processors/xapi/transformer.py

```diff
@@ -102,6 +102,15 @@ def get_actor(self):
             )
         return agent
 
+    @property
+    @openedx_filter(filter_type="event_routing_backends.processors.xapi.transformer.xapi_transformer.get_verb")
+    def verb(self):
+        """
+        This intercepts the super verb value or the attribute class `_verb` in order to allow the openedx
+        filters implementation.
+        """
+        return super().verb if hasattr(super(), "verb") else self._verb
```

Comment:
I am not clear with this line. So if the parent class has a verb could use that verb, but it doesn't have it would use the custom `_verb` defined in this PR?